src/lib/remove-undefined.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Cleans an object of undefined values
+ */
+export default function removeUndefined(input = {}) {
```

Comment:
Do we want _just_ undefined? Or falsy values?

Reply:
in this case we want to strip undefined only. `null` `0` `false` are all valid _defined_ values.

In our case we were passing in `undefined`s that were overriding defined values, which isn't the desired behavior.

Reply:
got it!